src/commands/react-native/README.md

```diff
@@ -93,9 +102,79 @@ This allows Datadog to create proper URLs such as:
 | GitHub or GitLab  | https://\<repository-url\>/blob/\<commit-hash\>/\<tracked-file-path\>#L\<line\> |
 | Bitbucket | https://\<repository-url\>/src/\<commit-hash\>/\<tracked-file-path\>#lines-\<line\>  |
 
+
+### `xcode`
+
+This command can be called from an xcode build phase to execute the react-native bundle command and then upload the sourcemaps.
+The upload only happens when your target has a "Release" build configuration to prevent overwriting existing sourcemaps.
+
+This command can use the same environment variables as the `upload` command: `DATADOG_API_KEY` (required), `DATADOG_SITE` and `DATADOG_SOURCEMAP_INTAKE_URL`.
+
+Then, you need to get the location to your `node` and `yarn` binaries by running in a terminal:
+
+```bash
+$ which node #/path/to/node
+$ which yarn #/path/to/yarn
+```
+
+Then change the "Bundle React Native code and images" build phase in XCode (don't forget to change `/path/to/node` and `/path/to/yarn`):
```

Comment:
```suggestion
Change the "Bundle React Native code and images" build phase in XCode, including `/path/to/node` and `/path/to/yarn`.
```

Reply:
I think we can remove this sentence, it was an introduction to the next blocks, but I find it confusing as users might wonder whether they need to do an action at this point (they don't)